src/pages/Popup/components/Settings.jsx

```diff
@@ -196,6 +211,7 @@ class Settings extends Component {
             id="submit_button"
             disabled={this.state.buttonDisabled}
             variant="contained"
+            style={styleobj}
```

Comment:
You can directly use `style={this.state.background}` instead of creating a new variable

Reply:
Updated.